internal/services/appservice/linux_function_app_data_source.go

```diff
@@ -141,6 +142,12 @@ func (d LinuxFunctionAppDataSource) Attributes() map[string]*pluginsdk.Schema {
 			Computed: true,
 		},
 
+		"client_cert_exclusion_paths": {
```

Comment:
i think we should not be abbreviating certificate?
```suggestion
		"client_certificate_exclusion_paths": {
```